fhir-validation/src/main/java/org/linuxforhealth/fhir/validation/FHIRValidator.java

```diff
@@ -330,11 +330,12 @@ private void validate(FHIRPathResourceNode resourceNode) {
                 constraints.addAll(ProfileSupport.getConstraints(profiles, resourceType));
             }
 
-            // path (with array indices) -> Extension instance (which never have a url version suffix)
-            Map<String, Extension> pathToExtension = gatherInstanceExtensions(resourceNode);
             // path (with no array indices) -> Extension URLs (with no version suffix)
             Map<String, Set<String>> alreadyGeneratedExtensionConstraints = gatherExtensionConstraints(constraints);
 
+            // path (with array indices) -> Extension instance (which never have a url version suffix)
+            Map<String, Extension> pathToExtension = gatherInstanceExtensions(resourceNode);
```

Comment:
another minor formatting thing I had locally... the foreach on the next line references pathToExtension and so it made more sense for this to be the line which proceeds that